filebeat/module/haproxy/log/ingest/pipeline.json

```diff
@@ -40,9 +40,19 @@
             }
         },
         {
-            "geoip": {
+            "grok": {
                 "field": "haproxy.client.ip",
```

Comment:
So this assume in some cases this is actually not an ip?

Reply:
In HAProxy's case, they're too performance-conscious for this to be a name resolved from a reverse DNS query. However this can be populated with text if the connection is coming from a Unix socket LOL

Reply:
Unix socket info seems to be one more new twist on the IP field :-(